openstack/keppel/templates/configmap-anycast.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.keppel.anycast_domain_name }}
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: keppel-anycast-nginx
+
+data:
+  proxy.conf.template: |
+    {{- .Files.Get "files/proxy.conf.template" | nindent 4 }}
```

Comment:
Since a few values need to be interpolated, it probably makes sense to put the config file in here verbatim.